src/tests/ftest/pool/destroy.yaml

```diff
@@ -6,16 +6,13 @@ setup:
   start_servers_once: False
 server_config:
   name: daos_server
+  system_ram_reserved: 1
```

Comment:
Why is this 1 and not 2, like the other VM test yamls?

Reply:
I have changed all assignments to 1, done